src/wfs/src/main/java/org/geoserver/wfs/xml/FeatureTypeSchemaBuilder.java

```diff
@@ -1075,5 +1075,28 @@ protected XSDSchema getWfsSchema() {
         
     }
     
+    /**
+     * Derived from {@link GML32}, the only difference is that {@link #getWfsSchema()} is overridden
+     * to make it always return {@code null}.
+     *
+     * Useful when encoding DescribeFeatureType responses, as they don't need to import the WFS schema.
+     *
+     * @author Stefano Costa, GeoSolutions
```

Comment:
Usually we don't use the ``@author`` tag anymore, we have ``git blame`` now :stuck_out_tongue:.